ingestion_server/ingestion_server/indexer.py

```diff
@@ -213,6 +214,30 @@ def _index_table(self, table, dest_idx=None):
             )
             self.replicate(table, dest_idx, query)
 
+    def _bulk_upload(self, es_batch):
+        max_attempts = 4
+        attempts = 0
+        # Initial time to wait between indexing attempts
+        # Grows exponentially
+        cooloff = 5
+        while True:
+            try:
+                deque(helpers.parallel_bulk(self.es, es_batch, chunk_size=400))
+            except elasticsearch.ElasticsearchException:
```

Comment:
Is `ElasticsearchException` the only exception that could be thrown in this case?

Reply:
A more specific exception would be better, but there is no way to predict what is being thrown until we run on a real dataset. We should be able to better narrow it down at a future date based on the information in `exc_info`.

Reply:
Right. I thought that maybe there could be a more general exception handler, but if not, then it's fine